gapis/api/gles/api/textures_and_samplers.api

```diff
@@ -440,9 +440,16 @@ sub void TexImage(TexImageFlags  flags,
     }
   }
 
+  // TODO(hevrard): check that if t.ImmutableFormat == GL_TRUE, we are
+  // not trying to attempt to modify the format of dimension of any
+  // level (see GLES spec '8.18 Immutable-Format Texture Images')
```

Comment:
you should say which version of the spec you are talking about, otherwise the section number may not match up. Typically we refer to the 3.2 spec.

Reply:
done